java/src/main/c++/vw_learner_VWScalarsLearner.h

```diff
@@ -0,0 +1,30 @@
+/* DO NOT EDIT THIS FILE - it is machine generated */
```

Comment:
Can we keep this out of the repository and machine generate on demand?  (What is the generation mechanism?)